### ISSUE TITLE
[Potential Issue] with RVN

### ISSUE THREAD

**Issue by setuidroot** [MoneroOcean/nodejs-pool#108]
I've noticed a potential issue with Raven (RVN) blocks not being mined (or not posting to the pool dashboard) properly.

Currently MoneroOcean has ~527% effort on RVN without a single block being mined.  This doesn't mean much, especially since miners mine RVN at low network difficulty (thus total RVN hashes might not be that high.)  Maybe bad luck.

Well, C3pool (your pool code) currently has ~2216% effort on RVN without a single block being mined.  MO + C3 = **really** bad luck?  🤔  This is unlikely.  I think there might be an issue.

Do you remember the problem with Loki where miner effort would go into thousands of percent without a block but eventually a bunch of blocks would post with 0% effort on the dashboard?  Maybe RVN is similar?  I don't know what caused that issue with Loki.  Maybe you had to manually reset it to get the mined blocks to show on the pool dashboard.  Maybe this is the case with RVN as well.  I don't know.  I just thought I'd bring this to your attention if you didn't already know.

**Comment by MoneroOcean** [comment]
Yeah, I'm monitoring RVN. Should be no issues to best of my knowledge. Will check logs for possible missing RVN blocks tomorrow.
